Specs/Sequence/Chinese/IpAddressModel.json

```diff
@@ -1,381 +1,363 @@
 [
-    {
-      "Input": "我电脑IP是1.1.1.1",
-      "NotSupported": "java",
-      "Results": [
-        {
-          "Text": "1.1.1.1",
-          "TypeName": "ip",
-          "Resolution": {
-            "value": "1.1.1.1",
-            "type": "ipv4"
-          }
+  {
+    "Input": "我电脑IP是1.1.1.1",
+    "NotSupported": "java",
```

Comment:
Why are these still disabled in Java only?

Reply:
BTW, please avoid changing line endings in spec files unless intentional. It makes review a bit harder.
I recommend reverting them.

Reply:
Hi @tellarin, we reverted the changes and disabled the ones that are not supported by Java.

The non-supported specs are because the Regex doesn't match with the text in Java. For example the input `我电脑IP是1.1.1.1` should be matched with the Regex `\b(1\d{2}|2[0-4]\d|25[0-5]|0?[1-9]\d|0{0,2}\d)((\.(1\d{2}|2[0-4]\d|25[0-5]|0?[1-9]\d|0{0,2}\d)){3})\b` but as you can see in the images it doesn't match using Java engine.

To validate this we used a JavaScript engine and Java engine with:
* [Freeformatter](https://www.freeformatter.com/java-regex-tester.html#ad-output): this is the Java RegEx tester
* [RegExr](https://regexr.com/): settings as JavaScript RegEx engine

Java doesn't match the text with the Regex
![image](https://user-images.githubusercontent.com/11904023/107517369-71933b80-6b8c-11eb-8b90-7727f9c4f2b6.png)

JavaScript matches the text with the Regex
![image](https://user-images.githubusercontent.com/11904023/107517378-73f59580-6b8c-11eb-8960-bb833a6fe0cd.png)